ui-admin/src/study/notifications/TriggerDesignerEditor.tsx

```diff
@@ -468,24 +469,28 @@ const TaskStatusEditor = (
   </div>
 }
 
-const TaskTargetStableIdsEditor = ({ studyEnvParams, stableIds, setStableIds }:
-  {studyEnvParams: StudyEnvParams, stableIds: string[], setStableIds: (ids: string[]) => void}) => {
-  const [surveys, setSurveys] = useState<Survey[]>([])
+const TaskTargetStableIdsEditor = ({ studyEnvParams, stableIds, setStableIds, isKitType }:
+  {studyEnvParams: StudyEnvParams, stableIds: string[], setStableIds: (ids: string[]) => void, isKitType: boolean}) => {
+  const [options, setOptions] = useState<{ label: string, value: string }[]>([])
   useLoadingEffect(async () => {
-    const studyEnvSurveys = await Api.findConfiguredSurveys(studyEnvParams.portalShortcode,
-      studyEnvParams.studyShortcode, studyEnvParams.envName, true)
-    setSurveys(studyEnvSurveys.map(studyEnvSurvey => studyEnvSurvey.survey))
-  })
+    if (isKitType) {
+      const kitTypes = await Api.fetchKitTypes(studyEnvParams)
+      setOptions(kitTypes.map(kitType => ({ label: kitType.displayName, value: kitType.name })))
+    } else {
+      const studyEnvSurveys = await Api.findConfiguredSurveys(studyEnvParams.portalShortcode,
+        studyEnvParams.studyShortcode, studyEnvParams.envName, true)
+      setOptions(studyEnvSurveys.map(ses => ({ label: ses.survey.name, value: ses.survey.stableId })))
+    }
+  }, [isKitType])
+
   stableIds = stableIds ?? []
   const inputId = useId()
   return <div className="mt-3">
-    <label className="form-label" htmlFor={inputId}>Limit to these surveys
+    <label className="form-label" htmlFor={inputId}>Limit to these {isKitType ? 'kit types' : 'surveys'}
       <span className="fst-italic ms-2">(leave blank if trigger applies to all)</span></label>
```

Comment:
```suggestion
      <span className="fst-italic ms-2">(leave blank if automation applies to all)</span></label>
```
Super minor thing since we aren't using the word `trigger` anywehre else in the UI